.travis.yml

```diff
@@ -10,18 +10,18 @@
 
 language: node_js
 os:
+  - windows
```

Comment:
This repo does not need Windows support as it never runs on Windows (as part of aio cli etc.).